services/submission/src/main/java/app/coronawarn/server/services/submission/validation/ValidSubmissionPayload.java

```diff
@@ -86,11 +87,14 @@ public SubmissionPayloadValidator(SubmissionServiceConfig submissionServiceConfi
     public boolean isValid(SubmissionPayload submissionPayload, ConstraintValidatorContext validatorContext) {
       List<TemporaryExposureKey> exposureKeys = submissionPayload.getKeysList();
       validatorContext.disableDefaultConstraintViolation();
```

Comment:
Consider adding a test which ensures that all rolling start interval numbers are at 00:00 UTC. This way, the `checkNoOverlapsInTimeWindow` test can be skipped for TEK sets without flexible rolling periods.